src/components/skybox.js

```diff
@@ -280,6 +288,20 @@ AFRAME.registerComponent("skybox", {
 
       this.el.object3D.matrixNeedsUpdate = true;
     }
+
+    this.updateEnvironmentMap();
+  },
+
+  updateEnvironmentMap() {
+    this.skyScene.add(this.sky);
+    this.cubeCamera.update(this.el.sceneEl.renderer, this.skyScene);
+    this.el.setObject3D("mesh", this.sky);
```

Comment:
This already happens in `init`, why does it need to get set again here?

Reply:
When I call `this.skyScene.add(this.sky)` it is removed from the main scene. The `setObject3D` call adds it back to the main scene.